src/extender.js

```diff
@@ -16,7 +17,9 @@ export const hook = (module, funcName, options = {}, shimmerLib = shimmer) => {
       if (isFunctionAlreadyWrapped(originalFn)) return originalFn;
       return function (...args) {
         safeBeforeHook.call(this, args, extenderContext);
+        Timer.pause();
         const originalFnResult = originalFn.apply(this, args);
+        Timer.start();
```

Comment:
If we using the decorators, this one should be removed?

Reply:
we can use then only on class methods